cpu/stm32f4/include/periph_cpu.h

```diff
@@ -102,6 +102,25 @@ typedef enum {
 /** @} */
 
 /**
+ * @brief   Calculate the I2C speed value
+ */
+#define I2C_SPEED(mode)         ((mode << 15) | (CLOCK_APB1 / (2 << (2 * mode) * 10000))
+
+/**
+ * @brief   Override I2C speed options
+ * @{
+ */
+#define HAVE_I2C_SPEED_T
+typedef enum {
+    I2C_SPEED_LOW       = -1,   /**< low speed mode:    ~10kbit/s */
+    I2C_SPEED_NORMAL    =  0,   /**< normal mode:       ~100kbit/s */
+    I2C_SPEED_FAST      =  1,   /**< fast mode:         ~400kbit/sj */
+    I2C_SPEED_FAST_PLUS = -1,   /**< fast plus mode:    ~1Mbit/s */
```

Comment:
Shouldn't we keep this as `1` as is the highest speed available? (same line below)